TourMate/TourMate/Frontend/Plan/Views/PlanViews/PlanDetailsViews/PlanView.swift

```diff
@@ -55,13 +55,11 @@ struct PlanView: View {
                         VStack(alignment: .leading) {
                             Text("Start Location")
                                 .font(.caption)
-                            Text(plan.startLocation)
+                            Text(plan.startLocation?.addressFull ?? "No start location provided")
 
-                            if let endLocation = plan.endLocation {
-                                Text("End Location")
-                                    .font(.caption)
-                                Text(endLocation)
-                            }
+                            Text("End Location")
+                                .font(.caption)
+                            Text(plan.endLocation?.addressFull ?? "No end location provided")
```

Comment:
This does not appear for me though

<img width="488" alt="image" src="https://user-images.githubusercontent.com/65328119/161249818-a019850f-bb36-45e6-9fca-2c7a70472b8a.png">

Reply:
Oh yes yes cause the `plan.startLocation` do exists. I'll think about how to show the location visually when I'm doing the map view. Will merge this PR first